src/cli/dev.js

```diff
@@ -6,7 +6,7 @@
  * Side Public License, v 1.
  */
 
-require('./apm')(process.env.ELASTIC_APM_SERVICE_NAME || 'kibana-proxy');
+require('./apm')(process.env.ELASTIC_APM_SERVICE_NAME || 'kibana');
```

Comment:
Doesn't it mean that APM will still report the same metrics twice in `dev` mode? There are still two processes but with the same name from now on. 
Maybe we should instrument `apm` agent only when `process.env.ELASTIC_APM_SERVICE_NAME === 'kibana'`?

Reply:
Can you please help me to find out where are the duplicate metrics? I think I'm missing something https://kibana-ops-e2e-perf.kb.us-central1.gcp.cloud.es.io:9243/app/apm/services/kibana-proxy/transactions?comparisonEnabled=true&comparisonType=period&environment=ENVIRONMENT_ALL&kuery=labels.testJobId%20%3A%20%22local-c0da3159-d6be-4e58-8680-5c0aad948c06%22&latencyAggregationType=avg&rangeFrom=now-10w&rangeTo=now&transactionType=request